core/src/main/scala/org/http4s/package.scala

```diff
@@ -136,4 +77,13 @@ package object http4s { // scalastyle:ignore
       case Left(l) => f(l)
     }
   }
+
+  implicit def http4sHttpServiceSemigroup[F[_]: Monad]: Semigroup[HttpService[F]] =
```

Comment:
This is just here to make stuff compile.

Reply:
Are we aiming to provide a `Semigroup` or a `SemigroupK`?

Reply:
In the specific case of `HttpService`, I guess a `Semigroup` works just as well.

This still needs to be manually in scope to work, I just wanted the rest to compile while we figure out what to do.